cmd/crc/cmd/start.go

```diff
@@ -154,3 +154,10 @@ func getPullSecretFileContent() (string, error) {
 
 	return pullsecret, nil
 }
+
+func getBundlePath() string {
+	if crcConfig.GetString(config.Bundle.Name) == "" {
+		return constants.DefaultBundlePath
+	}
+	return crcConfig.GetString(config.Bundle.Name)
+}
```

Comment:
Is this going to give us a different behaviour than what we had before? Instead of specifying a default of DefaultBundlePath, you use "" as the default, but you also make sure that each time we use the config value, "" is replaced with DefaultBundlePath, so in the end it's the same as if we had set the default as DefautBundlePath?

Reply:
Right, this change is not needed, just the other change is enough!